### PR TITLE
Moved StackGS to Legacy Docs

### PR DESCRIPTION
**Pull request by debadair**
Preview: https://docs_2459.docs-preview.app.elstc.co/guide/index.html

### CODE REVIEW

File: conf.yaml [hunk 1]
```diff
@@ -2473,6 +2448,32 @@ contents:
 
     -   title:      Legacy Documentation
         sections:
+          - title:      Getting Started
+            prefix:     en/elastic-stack-get-started
+            current:    8.2
+            index:      docs/en/getting-started/index.asciidoc
+            branches:   [ {main: master}, 8.3, 8.2, 8.1, 8.0, 7.17, 7.16, 7.15, 7.14, 7.13, 7.12, 7.11, 7.10, 7.9, 7.8, 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3 ]
```
Comment (gtback) [inline conf.yaml:2455]: Do we still want to publish main/master and 8.3 versions of this book? Especially if we "freeze" the live versions at 8.2 and 7.17?

**Review by gtback — APPROVED** [review]

**Comment by debadair** [comment]
@elasticmachine retest this please

**Comment by debadair** [comment]
@elasticmachine retest this please

**Comment by debadair** [comment]
@elasticmachine retest this please

File: conf.yaml (hunk 1 above)
Comment (debadair) [reply conf.yaml]: Nope. Fixed that. :-)

**Comment by debadair** [comment]
@elasticmachine retest this please

**Comment by debadair** [comment]
@elasticmachine retest this please

**Comment by debadair** [comment]
@elasticmachine retest this please

**Comment by debadair** [comment]
@elasticmachine retest this please

**Comment by gtback** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by lcawl** [comment]
I think this error:
> 12:55:13 INFO:build_docs:Bad cross-document links:
...
12:55:13 INFO:build_docs:  Kibana [8.3]: packages/kbn-doc-links/src/get_doc_links.ts contains broken links to:
12:55:13 INFO:build_docs:   - en/elastic-stack-get-started/8.3/index.html
12:55:13 INFO:build_docs:  Kibana [master]: packages/kbn-doc-links/src/get_doc_links.ts contains broken links to:
12:55:13 INFO:build_docs:   - en/elastic-stack-get-started/master/index.html
12:55:13 runbld>>> <<<<<<<<<<<< SCRIPT EXECUTION END <<<<<<<<<<<<

... is related to this part of the build:
https://github.com/elastic/docs/blob/c6c8068ebe0bdc9cee341800fcb1e31eeac6c407/build_docs.pl#L385

I've therefore drafted https://github.com/elastic/kibana/pull/135381 to remove the need for that `STACK_GETTING_STARTED` variable in 8.3+

**Comment by debadair** [comment]
Thanks @lcawl !

**Comment by debadair** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by debadair** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by gtback** [comment]
> ... is related to this part of the build:

🤦🏻 Thanks @lcawl, I should have realized this was the problem 😞

**Pull request merged**